### DISCUSSION TITLE
v7.5.0-preview.1 Release of PowerShell

### DISCUSSION THREAD

**Discussion by jeffshumphreys** [PowerShell/PowerShell#21123] [General]
This seems like more of a 7.4.2? Recently I'm using PowerShell heavily and so I'm tracking minor releases closely, so I don't know if this is commonly how much change comes in a dot release, or if it's a matter of the new year.

Is there any way to see the enhancements separate?  I've perused the list, and I'm not sure if there's a reason to update?  I'm fine with not updating, but I'd hate to miss a juicy new feature.

**Comment by 237dmitry** [comment]
> I'm fine with not updating, but I'd hate to miss a juicy new feature.

All changes are written on the release page. To update or not is more a matter of convenience and habit. For example, on Windows I do not update on intermediate releases, but on Linux I update on every release. If we talk about any breaking changes, I didn’t see them, maybe I just didn’t notice them, since they don’t appear in any way in my use of PowerShell.

But with every release the shell works faster and with lesser errors, this is my opinion.

**Comment by MartinGC94** [comment]
That's pretty normal. Features/fixes get added throughout the year so at the end of the year when it gets officially released you will probably think it's more worthy of the version bump.
